source/user-manual/files-backup/wazuh-central-components.rst

```diff
@@ -0,0 +1,143 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Learn how to keep a backup of key files of your Wazuh central components installation.
+  
+Wazuh central components
+========================
+
+To create a backup of the central components of your Wazuh installation, follow these steps. Repeat them on every cluster node you want to back up. 
+
+.. note::
+
+   You need root user privileges to execute the commands below.
+
+Preparing the backup
+--------------------
+
+#. Create the destination folder to store the files. For version control, add the date and time of the backup to the name of the folder.
+
+   .. code-block:: console
+
+      # bkp_folder=~/wazuh_files_backup/$(date +%F_%H:%M)
+      # mkdir -p $bkp_folder && echo $bkp_folder
+
+#. Save the host information.
+
+   .. code-block:: console
+
+      # cat /etc/*release* > $bkp_folder/host-info.txt
+      # echo -e "\n$(hostname): $(hostname -I)" >> $bkp_folder/host-info.txt
+
+Backing up the Wazuh server
+---------------------------
+
+#. Back up the Wazuh server data, certificates, and configuration files.
+
+   .. code-block:: console
+
+      # rsync -aREz \
+      /etc/filebeat/ \
+      /etc/postfix/ \
+      /var/ossec/api/configuration/ \
+      /var/ossec/etc/client.keys \
+      /var/ossec/etc/sslmanager* \
+      /var/ossec/etc/*.pem \
+      /var/ossec/etc/ossec.conf \
+      /var/ossec/etc/internal_options.conf \
+      /var/ossec/etc/local_internal_options.conf \
+      /var/ossec/etc/rules/local_rules.xml \
+      /var/ossec/etc/decoders/local_decoder.xml \
+      /var/ossec/etc/shared/ \
+      /var/ossec/logs/ \
+      /var/ossec/queue/agent-groups/ \
+      /var/ossec/queue/agentless/ \
+      /var/ossec/queue/agents-timestamp \
+      /var/ossec/queue/fts/ \
+      /var/ossec/queue/rids/ \
+      /var/ossec/stats/ \
+      /var/ossec/var/db/agents/ \
+      /var/ossec/var/multigroups/ $bkp_folder
+
```

Comment:
@s-ocando

Yes this file could also be relevant to keep a backup as complete as possible.
One thing to keep in mind, is that we should add a warning in case users have modified the default paths of any of these files. In that case they should be the ones to make sure to keep them.